cjworkbench/settings.py

```diff
@@ -264,12 +266,15 @@
 # Internationalization
 # https://docs.djangoproject.com/en/1.10/topics/i18n/
 
-LANGUAGE_CODE = "en-us"
+LANGUAGE_CODE = "en"
```

Comment:
This is ok for now; before we go public with i18n we should change this back to `en-us` in Python and call it `en_US` in JavaScript.